Sources/App/Views/PackageController/PackageReadme+Model.swift

```diff
@@ -124,6 +125,22 @@ extension Element {
         }
     }
 
+    func fixProtectedCachedImages() {
+        do {
+            let imageElements = try select("img[data-canonical-src]")
+            for imageElement in imageElements {
+                let cachedUrl = try imageElement.attr("src")
```

Comment:
Did you see this warning?

```
initialization of immutable value 'cachedUrl' was never used; consider replacing with assignment to '_' or removing it
```

Reply:
```suggestion
```

Reply:
I didn't, and I'm usually pretty sensitive to warnings. There was another line of code that validated the cached URL too but I removed it as it was unnecessarily strict. Fixed.